ui/src/common/components/Author/__tests__/__snapshots__/Author.test.jsx.snap

```diff
@@ -0,0 +1,279 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Author renders first_name and last_name 1`] = `
+<div
+  className="di"
+>
+  <UnlinkedAuthor
+    author={
+      Immutable.Map {
+        "full_name": "Name, Full",
+        "first_name": "Full",
+        "last_name": "Name",
+      }
+    }
+  />
+</div>
+`;
+
+exports[`Author renders first_name and last_name of editor 1`] = `
+<div
+  className="di"
+>
+  <UnlinkedAuthor
+    author={
+      Immutable.Map {
+        "full_name": "Name, Full",
+        "first_name": "Full",
+        "last_name": "Name",
+        "inspire_roles": Immutable.List [
+          "editor",
+        ],
+        "affiliations": Immutable.List [
+          Immutable.Map {
+            "value": "Affiliation",
+          },
+        ],
+      }
+    }
+  />
+  <span
+    className="pl1 secondary-color"
+  >
+    (
+    <AuthorAffiliationList
+      affiliations={
+        Immutable.List [
+          "Affiliation",
+        ]
+      }
+    />
+    )
+  </span>
+  <Tooltip
+    arrowPointAtCenter={false}
+    autoAdjustOverflow={true}
+    mouseEnterDelay={0.1}
+    mouseLeaveDelay={0.1}
+    placement="top"
+    title="editor"
+    transitionName="zoom-big-fast"
+  >
+    (ed.)
+  </Tooltip>
+</div>
+`;
+
+exports[`Author renders first_name and last_name with affiliations 1`] = `
+<div
+  className="di"
+>
+  <UnlinkedAuthor
+    author={
+      Immutable.Map {
+        "full_name": "Name, Full",
+        "first_name": "Full",
+        "last_name": "Name",
+        "affiliations": Immutable.List [
+          Immutable.Map {
+            "value": "Affiliation",
+          },
+        ],
+      }
+    }
+  />
+  <span
+    className="pl1 secondary-color"
+  >
+    (
+    <AuthorAffiliationList
+      affiliations={
+        Immutable.List [
+          "Affiliation",
+        ]
+      }
+    />
+    )
+  </span>
+</div>
+`;
+
+exports[`Author renders first_name with last_name missing 1`] = `
+<div
+  className="di"
+>
+  <UnlinkedAuthor
+    author={
+      Immutable.Map {
+        "full_name": "Name Full",
+        "first_name": "Name",
+        "affiliations": Immutable.List [
+          Immutable.Map {
+            "value": "Affiliation",
+          },
+        ],
+      }
+    }
+  />
+  <span
+    className="pl1 secondary-color"
+  >
+    (
+    <AuthorAffiliationList
+      affiliations={
+        Immutable.List [
+          "Affiliation",
+        ]
+      }
+    />
+    )
+  </span>
+</div>
+`;
+
+exports[`Author renders full_name if first_name and last_name missing 1`] = `
```

Comment:
this test doesn't make sense now.

Reply:
there are few like this, you need to remove these and test these cases in `getAuthorName` tests.